src/Components/FlashBanner.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react"
+import { Banner, Sans } from "@artsy/palette"
+
+interface Props {
+  messageCode: string
+}
+
+const textMessage: (msg: string) => React.FC = (message: string) => () => (
+  <Sans color="white100" size="3" lineHeight={1} weight="medium">
+    {message}
+  </Sans>
+)
+
+const messages: { [k: string]: React.FC } = {
+  confirmed: textMessage("Your email has been confirmed."),
+  // TODO: I don't think we should show this as a fallback as any string could trigger it
+  fallback: textMessage("An error has occurred. Please try again."),
```

Comment:
agreed with this TODO – I don't think we should fallback like this. Can we remove and enumerate through the states that Gravity expresses via URL parameters (described above).

Reply:
should we log or send a message to sentry for unhandled messages? i think not.